src/yotpo.js

```diff
@@ -1,4 +1,6 @@
+import axios from "axios";
 import request from "request";
```

Comment:
A little quick and dirty here, I didn't move the other requests away from request to axios, but I will as part of the followup ticket with the automated task.